### DISCUSSION TITLE
Maintainers

### DISCUSSION THREAD

**Discussion by BlueDrink9** [autokey/autokey#593] [Maintenance goals]
Here are two initiatives for helping struggling OSS projects find maintainers.

* [Jazzband](https://jazzband.co/about)
* [Adoptoposs](https://adoptoposs.org/)

Autokey desperately needs extra maintenance help, so it might be worth investigating putting ourselves on those services.

**Comment by josephj11** [comment]
Jazzband has a lot of users, but is focused almost exclusively on django. They know Python, but would they be interested in something like AutoKey?

Adoptoposs is more diverse, but smaller. Looks like a better place to start.

Both are pretty small, so I wouldn't expect an overwhelming response.

Since most of the responses will probably require technical expertise, our developers would have to be the ones to manage this. Getting help takes time too and someone will have to be available.

**Comment by sebastiansam55** [comment]
I messaged the official python discord and requested that we be added as a channel to their open source section, I'll update if they get back to me.

**Comment by Elliria** [comment]
It's also possible that, like me, many people who read this sort of thing aren't yet good enough at coding to be of proper use for development. I hope we find some interested developers with this, though. This is a project worth working on.
